gravityflow/gravityflow
# Fix non-editable fields required by GP Limit Dates not being retained

## Description
re: [HS#13891](https://secure.helpscout.net/conversation/1194802467/13891)

This is alternative approach to #336 

It repurposes the calculation dependencies feature so it can also be used by other scenarios where dependent fields are used, in this case date fields using the min and max date range features provided by GP Limit Dates.

## Testing instructions
- Import test form
- Submit form
- On the user input step find that the datepicker min and max range is not limited as expected
- Switch to this branch
- Reload UI step and find that the datepicker min and max range is now limited as expected

## Automated Test Enhancements
Added test `0044DateFieldsGFMergeTagCept` which uses `GFMergeTag` to test that the values of non-editable date fields required by GP Limit Dates are accessible.

## Documentation Changes?
N/A

## Checklist:
